frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -42,7 +67,14 @@ export const supportLogic = kea<supportLogicType>([
             },
         },
     })),
-    listeners(({}) => ({
+    listeners(({ actions }) => ({
+        openSupportForm: async ({ kind, target_area }) => {
+            actions.resetSendSupportRequest({
+                kind: kind ? kind : undefined,
+                target_area: target_area ? target_area : undefined,
```

Comment:
help 🙏  VS code shows a type error here
<img width="729" alt="Screenshot 2023-04-14 at 15 22 11" src="https://user-images.githubusercontent.com/890921/232055424-70d9d019-e25d-4c2b-8c91-5567651bbc6b.png">

Reply:
You'll want to allow `supportTicketKind` to be `null`, and when it's `null` then don't have a value and make sure it shows the placeholder.

Reply:
`export type supportTicketKind = 'bug' | 'feedback' | null`

Though this feels like it could be reused so should maybe go in `types.ts`

Reply:
any reason not to do `'bug' | 'feedback' | undefined` when should I use `null` vs `undefined`?

Reply:
it worked with null and the placeholder did show up

Reply:
You could do undefined!

Reply:
it didn't work for some reason, not sure why exactly :D